src/app/my-donations/my-donations.component.ts

```diff
@@ -33,7 +33,7 @@ export class MyDonationsComponent implements OnInit{
 
   ngOnInit() {
     this.pageMeta.setCommon(
-      'Big Give - Your Donation History', '', null
+      'Your Donation History', 'Your historic Big Give donations', null
```

Comment:
```suggestion
      'Your Donation History', 'Your Big Give donations', null
```

I think `historic` mostly means something important in history.